public_html/templates/base.html

```diff
@@ -98,21 +97,21 @@
 
 	<section class="records">
 		{% block rowheader %}
-			<div class="header-div container-fluid">
+			<header class="header-div container-fluid">
 				<div class="header-col col-sm-3 text-center">Page</div>
 				<div class="header-col col-sm-1 text-center">Diff</div>
 				<div class="header-col col-sm-2 text-center">Editor</div>
 				<div class="header-col col-sm-3 text-center">Wikiprojects</div>
 				<div class="header-col col-sm-2 text-center">Review</div>
-			</div>
+			</header>
 		{% endblock %}
-		<div class="record-container">
+		<section class="record-container">
 			{% block records %}
 			{% endblock %}
-		</div>
+		</section>
 		<div class="load-more-div">
 			{% block loadmore %}
-				<button id="btn-load-more" class="btn btn-md btn-primary" onclick="loadMoreResults()">Load More</button>
+				<button id="btn-load-more" class="btn btn-md btn-primary js-load-more">Load More</button>
 			{% endblock %}
 		</div>
 	</section>
```

Comment:
This whole file is just wrapping the JS in `$(document).ready` and adding the necessary listeners at the top